scripts/sentinel_granule.sh

```diff
@@ -80,28 +89,42 @@ if [ "$fmask_valid" == "invalid" ] && [ "$cloud_cover_valid" == "invalid" ]; the
   echo "Fmask reports no clear pixels. Exiting now"
   exit 4
 fi
+rm fmask_out.txt
 
 fmask="${safegranuledir}/FMASK_DATA/${grandir_id}_Fmask4.tif"
 
-echo "Converting to flat binary"
+if [ "$exit_after_fmask" == "true" ]; then
+  echo "Copying fmask file"
+  mkdir -p "${outputdir}/${outputname}/"
+  #cp $fmask "${outputdir}/${outputname}/${outputname}_Fmask4.tif"
+  cp $fmask "${outputdir}/${outputname}/${granule}_Fmask${fmaskversion}.tif"
```

Comment:
Does `${fmaskversion}` actually get set here? I am probably reading incorrectly, but I see the variable defined within `sentinel.sh` but without being `export`-ed. Might want to double check this doesn't evaluate to nothing